### ISSUE TITLE
Can't build go-flutter, request contributing guide 

### ISSUE THREAD

**Issue by befovy** [go-flutter-desktop/go-flutter#302]
## Is your feature request related to a problem?

I have the readme in go-flutter and go-flutter/embedder

>  To build this package set the `CGO_LDFLAGS` and run `go build`. For example:
export CGO_LDFLAGS="-L/home/${HOME}/.cache/hover/engine/linux/"
go build
This works if [hover](https://github.com/go-flutter-desktop/hover) has cached the flutter engine for the local user.

But it not works, hover dose not cached engine.

hover doctor
```
14:35:36  › hover doctor
hover: Running on darwin
hover: Docker installed: true
hover: Sharing flutter version
Flutter 1.10.7 • channel beta • https://github.com/flutter/flutter.git
Framework • revision e70236e36c (5 周前) • 2019-10-02 09:32:30 -0700
Engine • revision 9e6314d348
Tools • Dart 2.6.0 (build 2.6.0-dev.0.0 1103600280)
hover: Flutter engine commit: https://github.com/flutter/engine/commit/9e6314d348
hover: Finding out the C compiler version
clang version 8.0.0 (tags/RELEASE_800/final)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /usr/local/opt/llvm/bin
```
## Describe the feature you'd like

A clear contributig guide

**Comment by pchampio** [comment]
You have to adapt the `export CGO_LDFLAGS` string to your os (see the linux at the end).
Hover should cache the engine in the `/home/${HOME}/.cache/hover/engine/` dir.
Is the engine cached `ls -R /home/${HOME}/.cache/hover/engine/` ?

While I don't have written a contributing guide, I have spend a lot of time writing a [wiki](https://github.com/go-flutter-desktop/go-flutter/wiki).
One of the available section is: [How to work on go-flutter](https://github.com/go-flutter-desktop/go-flutter/wiki/Setting-up-hover-to-use-a-home-made-version-of-'go-flutter').

And BTW, we don't recommend directly importing `go-flutter/embedder` in your code:
```
Note that this package does not promise a stable API: types, functions, methods may all change in a breaking way.
```

**Comment by pchampio** [comment]
I don't feel like writing a contributing.md file.
If you want to PR one, I'll merge it, but to me the wiki stands for itself.

**Comment by befovy** [comment]
Yeah, wiki is great.

But there's no `hover` folder in `${HOME}/.cache`

You said `/home/${HOME}/.cache/hover/engine/`. Is it a typo?
I hava nothing in `/home`

```sh
9:25:39  › pwd && ls -a
/home
.  ..

```

**Comment by pchampio** [comment]
fixed by #305
